src/jshint.js

```diff
@@ -863,7 +863,7 @@ var JSHINT = (function() {
       funct["(blockscope)"].stack();
       advance("let");
       advance("(");
-      state.syntax["let"].fud.call(state.syntax["let"].fud, false);
+      state.syntax["let"].fud.call(state.syntax["let"].fud);
```

Comment:
@rwaldron do you have any idea why we're calling it this way instead of `state.tokens.curr.fud(false);`?